src/layouts/docs/index.tsx

```diff
@@ -20,36 +22,45 @@ interface DocsLayoutProps {
   }
 }
 
-const DocsLayout: React.FC<DocsLayoutProps> = (props) => (
-  <BaseNewLayout>
-    <div className={s.body}>
+const DocsLayout: React.FC<DocsLayoutProps> = ({
+  productName,
+  navData,
+  backToLink,
+  headings,
+  breadcrumbLinks,
+  children,
+  githubFileUrl,
+}) => (
+  <SidebarSidecarLayout
```

Comment:
Have simplified `docs-layout` a bit by leveraging a new `sidebar-sidecar-layout` component, which handles the layout positioning concerns.